src/data/resolvers/motions.ts

```diff
@@ -478,6 +478,33 @@ export const motionsResolvers = ({
         return null;
       }
     },
+    async votingState(_, { colonyAddress, motionId }) {
+      try {
+        const votingReputationClient = await colonyManager.getClient(
+          ClientType.VotingReputationClient,
+          colonyAddress,
+        );
+        const {
+          skillRep,
+          repSubmitted,
+        } = await votingReputationClient.getMotion(motionId);
+        // eslint-disable-next-line max-len
+        const maxVoteFraction = await votingReputationClient.getMaxVoteFraction();
+
+        const threasholdValue = skillRep
+          .mul(maxVoteFraction)
+          .div(bigNumberify(10).pow(18));
```

Comment:
I think @ArmandoGraterol already added an until for this, so that we don't have to keep repeating this code.